src/components/DashBoard.js

```diff
@@ -93,7 +106,7 @@ const DashBoard = ({
 
   const backButton = showSiteDetail && (
     <ThemeProvider theme={theme.backButton}>
-      <ButtonStyle translateHover={true} boxShadow={true} onClick={backButtonHandler}>
+      <ButtonStyle translateHover={true} boxShadow={true} onClick={() => backButtonHandler()}>
```

Comment:
You're going to want to wrap that function call in a curly brackets. Remember the following:

```js() => backButtonHandler()```

is the same as 
```js
function () {
  return backButtonHandler() 
}
```

Where 
```
() => { backButtonHandler() }
```

That's the same as 
```
function () {
  backButtonHandler()
}
```